apps/admin-x-activitypub/src/views/Explore/Explore.tsx

```
@@ -66,21 +66,26 @@ export const ExploreProfile: React.FC<ExploreProfileProps & {
-                            <Skeleton className='w-12' />
+                            <Skeleton className='w-24' />
                         </div>
                     }
                 </div>
-                {profile.bio &&
+                {isLoading ?
+                    <Skeleton className='w-full max-w-96' />
+                    :
+                    profile.bio &&
                     <div
                         dangerouslySetInnerHTML={{__html: profile.bio}}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)